src/components/ProjectCard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Card, Button } from 'flotiq-components-react';
+
+const ProjectCard = ({ onClick, name, excerpt, image }) => (
+    <Card
+        bordered={false}
+        horizontal
+        rounded="none"
+        onClick={onClick}
+        additionalClasses={['flex mb-5 !bg-transparent']}
+    >
+        <Card.Img
+            src={image}
+            alt={name}
+            additionalContainerClasses={['basis-full md:basis-1/2 xl:basis-3/5']}
+            additionalClasses={['w-full rounded-3xl']}
+        />
+        <Card.Body additionalClasses={['basis-full md:basis-1/2 xl:basis-2/5']}>
+            <div>
+                <Card.Title additionalClasses={['font-archivo uppercase tracking-widest']}>
+                    {name}
+                </Card.Title>
+                <Card.Text additionalClasses={['']}>
+                    <div
+                        dangerouslySetInnerHTML={{ __html: excerpt }}
+                        className="line-clamp-5 lg:line-clamp-3 lg:line-clamp-4 font-lora text-sm
+                        xl:text-base uppercase italic mb-10 lg:mb-5 xl:mb-10"
+                    />
+                </Card.Text>
+                <div className="relative mr-5 w-fit">
+                    <Button
+                        label="Explore"
+                        size="lg"
+                        additionalClasses={['bg-secondary font-archivo font-normal '
```

Comment:
use variant="secondary" instead of bg-secondary

Reply:
variant used